discovery/consul/consul.go

```diff
@@ -134,6 +152,9 @@ type SDConfig struct {
 	// Desired node metadata.
 	NodeMeta map[string]string `yaml:"node_meta,omitempty"`
 
+	// Sets if this should also emit a target per exposed path
+	FetchExposedPaths bool `yaml:"fetch_exposed_paths,omitempty"`
```

Comment:
Should we have a role `service` or `service_exposed_paths` to choose one behaviour or the other?

Reply:
I already answer this in [here](https://github.com/prometheus/prometheus/pull/7800#discussion_r601620763)